### PR TITLE
Reexport System.Logger from System.Logger.Extended

### PR DESCRIPTION
**Pull request by neongreen**

### CODE REVIEW

**Comment by neongreen** [comment]
Note to self: don't merge before https://github.com/wireapp/wire-server/pull/654

File: services/spar/src/Spar/Run.hs [hunk 1]
```diff
@@ -39,7 +39,6 @@ import qualified Network.Wai.Middleware.Prometheus as Promth
 import qualified Network.Wai.Utilities.Server as WU
 import qualified SAML2.WebSSO as SAML
 import qualified Spar.Data as Data
-import qualified System.Logger as Log
 import qualified System.Logger.Extended as Log
```
Comment (fisx) [inline services/spar/src/Spar/Run.hs]: What about re-exporting `Extended` as well?  (I have no opinion.)

Comment (neongreen) [reply services/spar/src/Spar/Run.hs]: Re-exporting `Extended` from where?

Comment (fisx) [reply services/spar/src/Spar/Run.hs]: From `System.Logger.Extended`, like `System.Logger`.

**Pull request merged**